libbeat/processors/add_cloud_metadata/provider_aws_ec2.go

```diff
@@ -0,0 +1,24 @@
+package actions
```

Comment:
Change is made, also applied on `add_locale` package too.
BTW, could the reviewdog be taught to recognize such convention?